token/js/client/token.js

```diff
@@ -764,13 +768,13 @@ export class Token {
   ): Promise<AccountInfo> {
     const info = await this.connection.getAccountInfo(account, commitment);
     if (info === null) {
-      throw new Error(FAILED_TO_FIND_ACCOUNT);
+      throw new Error(formatDynamicErrorMessage(FAILED_TO_FIND_ACCOUNT, account.toString());
```

Comment:
@jstarry these days, what's most fashionable way of structured error in js land? Still, these adhoc way are rampant?

Reply:
for two things, I want to add some `error.code = FAILED_...` and `error.context = {address}`.. For internal use.

Reply:
You can use the following syntax with backticks: `` `Failed to find account with address ${account.toString()}` ``

Reply:
I will have to do some research, this is something that could be improved in the web3 library as well!